Tests/ProcessLib/LiquidFlow/TestLiquidFlowMaterialProperties.cpp

```diff
@@ -0,0 +1,122 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file   TestLiquidFlowMaterialProperties.cpp
+ *
+ * Created on August 18, 2016, 4:53 PM
+ */
+
+#include <gtest/gtest.h>
+
+#include "TestTools.h"
+
+#include "ProcessLib/LiquidFlow/LiquidFlowMaterialProperties.h"
+#include "MaterialLib/Fluid/FluidProperty.h"
+
+#include "MaterialLib/PorousMedium/Porosity/Porosity.h"
+#include "MaterialLib/PorousMedium/Storage/Storage.h"
+
+#include "MeshLib/Mesh.h"
+
+using namespace ProcessLib::LiquidFlow;
+using namespace MaterialLib::Fluid;
+using ArrayType = MaterialLib::Fluid::FluidProperty::ArrayType;
+using Matrix = Eigen::MatrixXd;
+
+TEST(ProcessLibLiquidFlow, checkLiquidFlowMaterialProperties)
+{
+    const char xml[] =
+        "<material_property>"
+        "    <fluid>"
+        "        <density>"
+        "           <type>LiquidDensity</type>"
+        "           <temperature0> 273.15 </temperature0>"
+        "           <p0> 1.e+5 </p0> "
+        "           <bulk_modulus> 2.15e+9 </bulk_modulus>"
+        "           <beta> 2.0e-4 </beta> "
+        "           <rho0>999.8</rho0>"
+        "        </density>"
+        "        <viscosity>"
+        "            <type>Vogels</type>"
+        "            <liquid_type>Water </liquid_type>"
+        "        </viscosity>"
+        "    </fluid>"
+        "    <porous_medium>"
+        "        <porous_medium>"
+        "            <permeability>"
+        "                <values>2.e-10 0. 0. 0. 3.e-10 0. 0. 0. 4.0e-10</values>"
+        "            </permeability>"
+        "            <porosity>"
+        "                <type>Constant</type>"
+        "                <value> 0.2 </value> "
+        "            </porosity>"
+        "            <storage>"
+        "                <type>Constant</type>"
+        "                <value> 1.e-4 </value>"
```

Comment:
As mentioned above, currently I only consider the patch-wise heterogeneous porous medium properties, which occur in most applications of subsurface modelling. If element-wise heterogeneous porous  medium property is used, we can easily added by using  MeshElementProperties.